VirtoCommerce.Platform.Web/Scripts/app/dynamicProperties/blades/propertyValue-list.js

```diff
@@ -39,6 +41,15 @@
         blade.isLoading = false;
     };
 
+    function refreshDynamicProperties(data)  { 
```

Comment:
This implementation looks like as `ad hoc`  solution, better to implement the propagation of the changes up to blades tree as it has done in the third version and in addition, needs to change dynamic properties binding, need to get dynamic properties meta information from API rather than  from context object

Reply:
made some chenges